.ci/gcb-generate-diffs-new.yml

```diff
@@ -359,4 +359,6 @@ availableSecrets:
     - versionName: projects/673497134629/secrets/ci-test-service-account/versions/latest
       env: GOOGLE_SERVICE_ACCOUNT
     - versionName: projects/673497134629/secrets/ci-test-service-account-key/versions/latest
-      env: SA_KEY
\ No newline at end of file
+      env: SA_KEY
+    - versionName: projects/673497134629/secrets/ci-test-public-advertised-prefix-description/versions/latest
+      env: GOOGLE_PUBLIC_AVERTISED_PREFIX_DESCRIPTION
```

Comment:
This env variable is needed for VCR test right? We probably need to pass it to https://github.com/GoogleCloudPlatform/magic-modules/blob/main/.ci/gcb-generate-diffs-new.yml#L243 as well